pythonFiles/conda_run_script.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+if __name__ != "__main__":
+    raise Exception("{} cannot be imported".format(__name__))
+
+import runpy
+import sys
+
+# Activating conda can print out stuff before the actual output is
+# printed. Hence, printing out markers to make it more resilient to
+# pull the output.
+print(">>>CONDARUNOUTPUT", end="")
```

Comment:
```suggestion
print(">>>CONDA-RUN-OUTPUT", end="")
```

Might as well make it readable.